README.md

```diff
@@ -0,0 +1,166 @@
+    <li><a href="">JAVASCRIPT</a></li>
+    <li><a href="">LINTERS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Add Book]**
+- **[Display books]**
+- **[Remove book]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://menendezon.github.io/awesome-module-books/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+This project doesn't need something to install, run this command to download the project : 
+git clone git@github.com:MenendezON/awesome-books.git
+
+### Prerequisites
+
+In order to run this project, you need :
+- Git to clone it (optional) 
+- A browser to run it
+- Some skill in HTML/CSS and js
+- An IDE (optional)
+
+### Install
+
+To install the project, run this command :
+**npm install**
+
+### Usage
+This project is only for creative commons, you can edit but don't forget to give credit back to the project developers.
+To run the project on your computer, execute the following command :
+**npm start**
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npx hint .
+```
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+
+### Deployment
+
+This project is deployed at github pages you can find it at: https://menendezon.github.io/awesome-books/
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **MENENDEZ NELSON**
+
+- GitHub: [@menendezon](https://github.com/menendezon)
+- Twitter: [@menendezon](https://twitter.com/menendezon)
+- LinkedIn: [MenendezON](https://linkedin.com/in/menendezon)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Edit book]**
+- [ ] **[Popup Page]**
+- [ ] **[Upload Image]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project Please drop a comment🤗
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[How will this project be of benefit to me?]**
+
+  - [This project will definitely help in storing the names, author and the description of your books free of charge]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENCE) licensed.
```

Comment:
- [ ] Please fix the LICENSE link the current link doesn't take you to the license file.